boards/shields/wuxi_ov5640/wuxi_ov5640.overlay

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/{
+	chosen {
+		zephyr,camera = &csi;
+	};
+};
+
+&lpi2c6 {
```

Comment:
In the past when defining shields for a NXP specific interface (which I believe this is?) we have named the interface something like `nxp_mipi_i2c` within the board devicetree- as this overlay is written, it will *only* work on the RT1170 EVK or any other EVK that happens to use LPI2C6. See here for an example: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/shields/rk055hdmipi4ma0/rk055hdmipi4ma0.overlay#L27

Reply:
In fact, lpi2c6 is used across all RT11XX family and I suppose this shield will work only for RT11XX family.
But yes, we should be generic. I will use some alias so that the shield is as generic as possible.